<a href="https://colab.research.google.com/github/jadynpeterson/physics/blob/master/Equation_Lab.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [12]:
import numpy as np

In [16]:
def rule1(dA):
    dQ=c(dA)
    return dQ

    dA=0.5

    dQ=rule1(dA)

    print ("dQ=",dQ)

In [21]:
def rule2(dA):
  dQ=cm(A**(m-1))(dA)
  return dQ

  dA= 0.5

  dQ= rule2(dA)

  print ("dQ=",dQ)

In [17]:
def rule3(dA, dB):
  dQ= np.sqrt(dA**2+dB**2)
  return dQ

  dA= 0.5
  dB=0.1

  dQ=rule3(dA, dB)

  print ("dQ=",dQ)

In [23]:
def rule4(dA, dB):
  dQ= np.sqrt((dA/A)**2(dB/B)**2)
  return dQ

  dA=0.5
  dB=0.1

  dQ=rule4(dA, dB)

  print("dQ=",dQ)

In [25]:
x = np.array([1.1,1.3,1.4,0.9,0.95,1.05])

print(np.std(x))
print(np.average(x))

0.17950549357115014
1.1166666666666667


The standard dev of the data set is 0.1795

The average of the data set is 1.117

# Rule 3

$\delta Q = \sqrt{(deltaA)^2+(deltaB)^2}**bold text**$

$\delta \Sigma Fc = {((4\pi)^2mr)/(T^2)}$